base/float.jl

```diff
@@ -1100,7 +1100,23 @@ floattype(::Type{Int16}) = Float16
 
 
 ## Array operations on floating point numbers ##
+"""
+    float(A::AbstractArray)
+
+Return an array containing the floating-point analog of each entry in array `A`.
 
+Equivalent to `float.(A)`, except that the return value may alias all or part
+of `A` in accordance with the behavior of `convert(T, A)` given output type `T`.
+
+# Examples
+```jldoctest
+julia> float([1, 2, 3])
+3-element Vector{Float64}:
+ 1.0
+ 2.0
+ 3.0
+```
```

Comment:
You should probably move the range example from the other float() docstring to here since `UnitRange <: AbstractArray`. The example in question is on lines 287-288.